ci/code_checks.sh

```diff
@@ -207,6 +207,14 @@ if [[ -z "$CHECK" || "$CHECK" == "patterns" ]]; then
     invgrep -R --include=*.{py,pyx} 'xrange' pandas
     RET=$(($RET + $?)) ; echo $MSG "DONE"
 
+    MSG='Check for use of ("%r" % foo) instead of {repr(foo)}' ; echo $MSG
+    invgrep -R --include=*.{py,pyx} -E '\s%r' pandas | invgrep -v '%%r'
```

Comment:
The `\s` seems arbitrary. If there are cases of `%r` that we are happy to have, I think we should not add this rule.

Also, I don't understand the piped `invgrep`, but this looks too complex.

Reply:
Agree, Will remove this rule.